Kernel/Arch/aarch64/RPi/Timer.cpp

```diff
@@ -35,7 +36,7 @@ Timer::Timer()
 
 Timer& Timer::the()
 {
-    static Timer instance;
+    static AK::Singleton<Timer> instance;
```

Comment:
Per discord, using AK::NeverDestroyed is a bit nicer: It also prevents the dtor call, while not requiring a heap allocation.

Reply:
Changed to use `AK::NeverDestroyed` instead!